applications/GeoMechanicsApplication/custom_elements/transient_Pw_interface_element.cpp

```diff
@@ -664,16 +665,17 @@ void TransientPwInterfaceElement<TDim, TNumNodes>::CalculateAndAddFluidBodyFlow(
 {
     KRATOS_TRY;
 
-    noalias(rVariables.PDimMatrix) = -PORE_PRESSURE_SIGN_FACTOR *
-                                     prod(rVariables.GradNpT, rVariables.LocalPermeabilityMatrix) *
-                                     rVariables.JointWidth * rVariables.IntegrationCoefficient;
+    const double&                          FluidDensity = this->GetProperties()[DENSITY_WATER];
```

Comment:
Since this is just a double, I would skip the `&` (copying is not more expensive, but clearer in this case). Also, this is a local variable, so in line with the Kratos Style Guide, we should use snake_case.
```suggestion
    const auto                          fluid_density = this->GetProperties()[DENSITY_WATER];
```

Reply:
changed